From f0e5910b4a8b689677e62af9efd98c42dacb0250 Mon Sep 17 00:00:00 2001
From: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
Date: Wed, 24 Apr 2024 15:41:07 -0300
Subject: [PATCH] csireverseproxy: fix config inclusion in configmap

This patch consists of changing the configmap
template to pass by helm lint.

Signed-off-by: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
---
 .../charts/csireverseproxy/templates/configmap.yaml             | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/charts/csi-powermax/charts/csireverseproxy/templates/configmap.yaml b/charts/csi-powermax/charts/csireverseproxy/templates/configmap.yaml
index 27938ea..135de43 100644
--- a/charts/csi-powermax/charts/csireverseproxy/templates/configmap.yaml
+++ b/charts/csi-powermax/charts/csireverseproxy/templates/configmap.yaml
@@ -4,4 +4,4 @@ metadata:
   name: {{ .Release.Name }}-reverseproxy-config
   namespace: {{ .Release.Namespace }}
 data:
-{{ tpl (.Files.Glob "conf/config.yaml").AsConfig . | indent 2 }}
+{{- tpl (.Files.Glob "conf/config.yaml").AsConfig . | nindent 2 }}
-- 
2.34.1

